docs/pages/introduction/faq.md

```diff
@@ -6,43 +6,103 @@ import { Collapsible } from '~/ui/components/Collapsible';
+
+Yes, the source for Expo Go can be found in the [expo/expo GitHub repository](https://github.com/expo/expo) in the `home`, `ios`, and `android` directories. The Expo Go app is also built with Expo and React Native!
+
+</Collapsible>
+
+<Collapsible summary="Who created Expo?">
+
+Expo and Expo Application Services are developed by [650 industries](https://expo.dev/about). 650 industries a is a California-based company that was founded by [Charlie Cheever](https://en.wikipedia.org/wiki/Charlie_Cheever) and [James Ide](https://jameside.com/).
+
+</Collapsible>
+
+<Collapsible summary="Why does Expo have its own SDK?">
+
+When Expo was first created, React Native had yet to be publicly released, meaning there were no third-party packages. In order to make the developer experience of React Native reasonable, we created a number of libraries for common functionality. Many of these libraries have since been forked and modified to meet various needs, we welcome users to mix and match whichever custom native code they need to make their app great.
+
+The Expo SDK is well tested, written in TypeScript, documented, and built for iOS, Android, and web. Every module in the Expo SDK works together to ensure versioning always matches, this creates a nice upgrading experience.
+
+The Expo SDK is also written with the [Expo native API](/modules) to make it easier to contribute, maintain, and understand.
+
+</Collapsible>
+
+<Collapsible summary="Can I use Expo with this web library?">
+
+Many popular web packages like three.js or Firebase work with Expo and React Native. Check out the [Expo examples](https://github.com/expo/examples) to learn more.
+
+</Collapsible>
+
+<Collapsible summary="Can I use Expo with my native library?">
+
+You can use native iOS and Android libraries with Expo by creating a [custom native module](/modules/) with Swift and Kotlin. Many popular libraries already have custom native modules, check out our [React Native directory](https://reactnative.directory/) to learn more.
+
+</Collapsible>
+
 <Collapsible summary="Is Expo similar to React for web development?">
 
-Expo and React Native are similar to React. You'll have to learn a new set of components (`View` instead of `div`, for example) and writing mobile apps is very different from websites; you think more in terms of screens and different navigators instead of separate web pages, but much more of your knowledge carries over than if you were writing a traditional Android or iOS app.
+Expo is a framework for React Native, which is a library for building iOS and Android apps. React Native is similar to `react-dom` for web development, enabling you to run React on a particular platform, but it has a few key differences:
+
+- React Native does not support HTML or CSS.
+- Instead of using the DOM, React Native uses native components. `<View />` instead of `<div />`, for example. Native components are more performant than the DOM and provide a much nicer user experience.
```

Comment:
```suggestion
- Instead of using the DOM, React Native uses native components. For example, `<View />` instead of `<div />`. Native components are more performant than the DOM and provide a much nicer user experience.
```